src/imitation/util/rollout.py

```diff
@@ -63,64 +47,36 @@ def step(self, action):
     return obs, rew, done, info
 
 
-def unwrap_traj(traj: Trajectory) -> Trajectory:
+def unwrap_traj(traj: data.TrajectoryWithRew) -> data.TrajectoryWithRew:
   """Uses `RolloutInfoWrapper`-captured `obs` and `rews` to replace fields.
 
   This can be useful for bypassing other wrappers to retrieve the original
   `obs` and `rews`.
 
-  Fails if `infos` is None or if the Trajectory was generated from an
+  Fails if `infos` is None or if the trajectory was generated from an
   environment without imitation.util.rollout.RolloutInfoWrapper
 
   Args:
-    traj: A Trajectory generated from `RolloutInfoWrapper`-wrapped Environments.
+    traj: A trajectory generated from `RolloutInfoWrapper`-wrapped Environments.
 
   Returns:
     A copy of `traj` with replaced `obs` and `rews` fields.
   """
   ep_info = traj.infos[-1]["rollout"]
-  res = traj._replace(obs=ep_info["obs"], rews=ep_info["rews"])
+  # no need to replace next_obs since that is never changed by VecEnv auto-reset
```

Comment:
I found this comment confusing because `TrajectoryWithRew` doesn't have `next_obs`.

Is the comment saying here that it may seem like there is an edge case involving terminal observations, but this is actually sidestepped?

Reply:
Hm, I think maybe I thought this was for transitions? I'll just delete this comment, I think it's confusing.